cdap-docs/reference-manual/source/http-restful-api-v3/introduction.rst

```diff
@@ -0,0 +1,179 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: HTTP RESTful Interface to the Cask Data Application Platform
+    :copyright: Copyright © 2014 Cask Data, Inc.
+
+.. _http-restful-api-introduction:
+
+===========================================================
+Introduction
+===========================================================
+
+.. highlight:: console
+
+.. _http-restful-api-conventions:
+
+Conventions
+============
+
+In this API, *client* refers to an external application that is calling CDAP using the HTTP interface.
+*Application* refers to a user Application that has been deployed into CDAP.
+
+.. rubric:: Base URL
+
+All URLs referenced in this API (with the exception of those in the 
+Namespace API) have this base URL::
+
+  http://<host>:<port>/v3/namespaces/<namespace-id>
```

Comment:
> under v2 should, if you replace /v2/ with /v3/namespaces/default/, it should still work. 
> That should be possible, no?

Its not always the case, since there are some APIs that are not *under* namespaces. System logs, services are examples. Like you said, ``/namespaces/system`` makes it better, but requires users to know about *system* vs *default*

I really like the idea of the summary page.